src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj

```diff
@@ -22,7 +22,6 @@
     <DevelopmentDependency>true</DevelopmentDependency>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
     <UsePublicApiAnalyzer>false</UsePublicApiAnalyzer>
-    <GetSigningInputsDependsOn>GetSigningInputsCustom</GetSigningInputsDependsOn>
```

Comment:
can we get some review pointers for how the assembly is picked now?
How do we point to the ilmerged nuget.exe as a target to sign?

Reply:
The individual assemblies are not picked any longer. That is why the vast majority of the signing infra was ripped out.

Instead, signtool is passed the list of top level artifacts to sign. Here; https://github.com/NuGet/NuGet.Client/pull/6539/files#diff-a86379b589f348a16f02f06fdb1518b40c78cc39ef21505d0134d06f86afb387R42-R44

It recursively unpacks all of these artifacts, applies the rules for what signatures are used when, signs, and repacks (and then signs the outer containers if necessary).

That means there's no sequencing required. Signing is moved after all packages are created.